gap_analysis/reports/business_report.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        
+    </data>
+</openerp>
```

Comment:
No need for an empty file nevertheless.
I could add that note to the Roadmap section.